doc/getting_started/build_app.md

```diff
@@ -2,174 +2,82 @@
 
 At this point you should have [set up your environment and installed Panel](installation.md) so you should be ready to get going.
 
-In this section we're going to be building a basic application using a public dataset and add some interactivity.
+On this page, we're going to be building a basic interactive application. If you want to implement this app yourself as you follow along, we recommend starting with a Jupyter notebook.
 
-:::{important}
-This guide renders static output by default (denoted by the golden border), but you can execute the code cells below by clicking the play button. Only run this if you are willing to download ~40 MB once. The download will be cached and reused the next time you click the play button on any page across the site.
-:::
+## Fetch the data
 
-## Fetch some data
-
-Panel lets you add interactive controls for just about anything you can display in Python. Panel can help you build simple interactive apps, complex multi-page dashboards, or anything in between. As a simple example, let's say we have loaded the [UCI ML dataset measuring the environment in a meeting room](http://archive.ics.uci.edu/ml/datasets/Occupancy+Detection+):
+First let's load the [UCI ML dataset](http://archive.ics.uci.edu/ml/datasets/Occupancy+Detection+) that measured the environment in a meeting room:
 
+:::{important}
+At first, this website renders code block outputs with limited interactivity, indicated by the <font color="darkgoldenrod">golden</font> border to the left of the output below. By clicking the play button (<svg class="pyodide-run-icon" style="width:32px;height:25px" viewBox="0 0 24 24"> <path stroke="none" fill="#28a745" d="M8,5.14V19.14L19,12.14L8,5.14Z"></path> </svg>) you can activate full interactivity, indicated by a <font color="green">green</font> left-border.
+:::
 
 ```{pyodide}
-import pandas as pd; import numpy as np; import matplotlib.pyplot as plt
+import panel as pn
+pn.extension()
+import hvplot.pandas
+import pandas as pd
+import numpy as np
```

Comment:
```suggestion
import panel as pn
import hvplot.pandas
import pandas as pd
import numpy as np

pn.extension(design='material')
```

I think it's okay to sneak this in here, it will make the app look better.